model/src/main/java/cz/muni/fi/pa165/team/match/containers/MatchListContainer.java

```diff
@@ -0,0 +1,61 @@
+package cz.muni.fi.pa165.team.match.containers;
+
+import cz.muni.fi.pa165.team.match.TeamMatch;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * @author Tomas Smid <smid.thomas@gmail.com>
+ */
+public class MatchListContainer
+{
+    private Collection<TeamMatch> matches;
+
+    private Map<TeamMatch, Long> homeGoals;
```

Comment:
Kdybys měl jednu strukturu na zápas a góly k němu

```java
public class MatchResult
{
    private TeamMatch match;
    private Long homeGoals;
    private Long awayGoals;
}
```

tak to daleko lépe vyjadřuje, co vlastně vracíš za hodnotu a co reálně budeš potřebovat (góly k tomu konkretnimu zapasu)